docs/tools/map.md

```diff
@@ -2,13 +2,18 @@
 
 The Map tool can be accessed inthe toolbar on the bottom right of the Game Board, but only in [Build mode](/docs/tools-overview/#tools-and-modes) and only by the GM.
 It offers an easy to use method to fit a custom shape to the grid.
-It can be used to resize arbitrary shapes, but its core use is focussed on maps.
+It can be used to resize arbitrary shapes, but its core use is focused on maps.
```

Comment:
Although technically my spelling wasn't incorrect, this is indeed way more common :)